ironfish-cli/src/commands/testnet.ts

```diff
@@ -0,0 +1,143 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { flags } from '@oclif/command'
+import cli from 'cli-ux'
+import { WebApi } from 'ironfish'
+import { IronfishCommand } from '../command'
+import { DataDirFlag, DataDirFlagKey, VerboseFlag, VerboseFlagKey } from '../flags'
+
+export default class Testnet extends IronfishCommand {
+  static hidden = false
+  static description = 'Set up your node to mine for the testnet'
+
+  static flags = {
+    [VerboseFlagKey]: VerboseFlag,
+    [DataDirFlagKey]: DataDirFlag,
+    confirm: flags.boolean({
+      default: false,
+      description: 'confirm without asking',
+    }),
+    skipName: flags.boolean({
+      default: false,
+      description: "Don't update your node name",
+    }),
+    skipGraffiti: flags.boolean({
+      default: false,
+      description: "Don't update your graffiti",
+    }),
+  }
+
+  static args = [
+    {
+      name: 'user',
+      required: false,
+      description:
+        'the user id or url to a testnet user like https://testnet.ironfish.network/users/1080',
+    },
+  ]
+
+  async start(): Promise<void> {
+    const { flags, args } = this.parse(Testnet)
+    let userArg = ((args.user as string | undefined) || '').trim()
+
+    if (!userArg) {
+      userArg = (await cli.prompt(
+        'Enter the user id or url to a testnet user like https://testnet.ironfish.network/users/1080\nUser ID or URL',
+        {
+          required: true,
+        },
+      )) as string
+      this.log('')
+    }
+
+    let userId: number | null = null
+
+    if (userArg.indexOf('/users/') !== -1) {
+      const index = userArg.indexOf('/users/')
+      const id = Number(userArg.slice(index + '/users/'.length))
+
+      if (!isNaN(id)) {
+        userId = id
+      }
+    } else if (!isNaN(Number(userArg))) {
+      userId = Number(userArg)
+    }
+
+    if (userId === null) {
+      this.log(`Could not figure out testnet user id from ${userArg}`)
+      this.exit(1)
+    }
+
+    // request user from API
+    this.log(`Asking Ironfish who user ${userId} is...`)
```

Comment:
Minor branding nit: Should be "Iron Fish".

Reply:
fixed